### DISCUSSION TITLE
Team Communication

### DISCUSSION THREAD

**Discussion by zachlankton** [mssiegel/Frempco-web-client#11] [Polls]
While I have started a discord server so that I can receive notifications in a way that makes me most productive, I would like to get a sense of what the preference of the current team would be for communication.  I've setup a simple poll to vote on what you would prefer!

Feel free to leave any comments or discuss other options!

-Zach

@mssiegel 
@karlrez 
@leahnp613

**Comment by leahnp613** [comment]
I'm team slack!

**Comment by mssiegel** [comment]
Thanks for setting up this poll!  I prefer Slack or WhatsApp.   I didn't know there was a Google Chat so I can't comment on it haha.

**Comment by karlrez** [comment]
I like whatsapp and slack!

**Comment by zachlankton** [comment]
Slack it is!  Ill get it setup!

**Comment by zachlankton** [comment]
https://join.slack.com/t/slack-fkf6136/shared_invite/zt-1mlqy0l7o-OlzpTRSaPk65iF863QwiLA

Use this link to join the slack channel!

@mssiegel 
@leahnp613 
@karlrez
